### DISCUSSION TITLE
Transfer of Tables from one node to another over SpaceWire?

### DISCUSSION THREAD

**Discussion by rlourette** [nasa/cFS#650] [Q&A]
I am considering writing a remote table app that will allow one to "open" tables on one CPU node and transfer them to the requesting node. I have SpaceWire in my system so RMAP can be used to physically move the data, but that is not applicable as a general solution.  Is there a more portable way of doing this perhaps CFDP based? I have several nodes (SpaceWire) in my system and need to send them tables from the main CPU. Thoughts?

**Comment by rlourette** [comment]
Anyone have any input?

**Reply by JiveyGuy** [reply]
How large are the tables?

**Reply by rlourette** [reply]
They could be pretty big. 100k ish
